x-pack/plugins/spaces/public/share_saved_objects_to_space/components/selectable_spaces_control.tsx

```diff
@@ -73,6 +74,19 @@ const APPEND_CANNOT_DESELECT = (
     type="iInCircle"
   />
 );
+const APPEND_PROHIBITED = (
+  <EuiIconTip
+    title={i18n.translate('xpack.spaces.shareToSpace.prohibitedSpaceTooltipTitle', {
+      defaultMessage: 'Cannot assign to this space',
+    })}
+    content={i18n.translate('xpack.spaces.shareToSpace.prohibitedSpaceTooltip', {
+      defaultMessage:
+        'A copy of this saved object exists in this space. In a future release, you can merge copies of saved objects.',
```

Comment:
Promising that it will be better in the future doesn't really help the user _today_. On the other hand it creates an expectation / obligation that we will provide this feature. I think it's better to try to come up with something that's actionable today.

Problem is what should a user really do in this situation? "To proceed delete object x from space y" will allow them to share but could lead to data loss e.g. a broken dashboard.

Can we allow users to "overwrite" the copy by deleting it and then proceeding to share the saved object? We need to have an url alias in place from the copy _id to the newly shared object.

Another potential resolution could be to drop the `originId` and say "I no longer care to restore this copy from backups" but this feels hard to communicate to a user and reason about.

I don't really know what's the solution, but at the moment it feels like we stop users from making things worse, but we don't really provide them with a way to proceed to do what they are trying to do except for "wait until a future release".

Reply:
I agree with Rudolf here. I don't think we should defer people to a future release. It doesn't really soften the blow to me to be honest, if anything it makes it worse. I think telling users that "A copy of the saved object already exists in this space" similar to the message below is completely acceptable. As a user I wouldn't assume that I should be able to assign it to that space if I've already got a copy there if you wouldn't have told me that a future release will allow you to do that. 

<img width="563" alt="Screenshot 2022-04-19 at 19 12 05" src="https://user-images.githubusercontent.com/190132/164069505-30dfda66-5bfa-4f4d-aae6-2597ff7f9fd2.png">